packages/ckeditor5-code-block/tests/codeblockediting.js

```diff
@@ -734,6 +735,26 @@ describe( 'CodeBlockEditing', () => {
 					return editor.destroy();
 				} );
 		} );
+
+		it( 'should convert markers inside pre > code', () => {
```

Comment:
Do we need this test? It looks the same as the test below (only it uses `markerToElement`).

Reply:
Wanted to cover all pipelines as other tests do. For editing downcast the right method is the `markerToElement`, but for data downcast it's `markerToData`. From my point of view it was a totally different case but I might be wrong here.